packages/fxa-settings/package.json

```diff
@@ -25,7 +25,7 @@
     "build": "INLINE_RUNTIME_CHUNK=false rescripts build",
     "test": "rescripts test",
     "eject": "react-scripts eject",
-    "lint": "npm-run-all --parallel lint:*",
+    "lint": "npm-run-all --parallel lint:eslint lint:sass",
```

Comment:
Not a huge deal, but why make all these an explicit list? For clarity's sake?

Reply:
something about the internal implementation of npm-run-all and yarn not liking the `*` 🤷